documentation/modules/auxiliary/scanner/snmp/snmp_enumusers.md

```diff
@@ -0,0 +1,33 @@
+## Description
+This module queries a range of hosts via SNMP and gathers a list of usernames on the remote system.
```

Comment:
I would mention this works on Windows and Sun (Solaris? @bcoles do you know if `sysDescr.0` on modern solaris still says "Sun"?  This module may need updating..)